src/javascript/ContentEditor/SelectorTypes/Picker/actions/openInTabAction.js

```diff
@@ -10,7 +10,7 @@ import {useSelector} from 'react-redux';
 
 export const OpenInTabActionComponent = ({render: Render, loading: Loading, path, field, inputContext, ...others}) => {
     const {lang} = useContentEditorContext();
-    const fallbackLanguage = useSelector(state => state.language);
+    const {fallbackLanguage, currentUILang} = useSelector(state => ({fallbackLanguage: state.language, currentUILang: state.uilang}));
```

Comment:
best practice to add `shallowEqual` option when returning objects in `useSelector`